### ISSUE TITLE
Prosody image build fails while patching muc.lib.lua

### ISSUE THREAD

**Issue by pandel** [jitsi/docker-jitsi-meet#1241]
Hi!

I tried to rebuild my images with the latest git version and got the following error during prosody image build:

```
The following NEW packages will be installed:
  jitsi-meet-prosody
0 upgraded, 1 newly installed, 0 to remove and 0 not upgraded.
Need to get 60.1 kB of archives.
After this operation, 321 kB of additional disk space will be used.
Get:1 https://download.jitsi.org stable/ jitsi-meet-prosody 1.0.5913-1 [60.1 kB]
Fetched 60.1 kB in 1s (60.7 kB/s)
Download complete and in download only mode
patching file muc.lib.lua
Hunk #1 FAILED at 1256.
1 out of 1 hunk FAILED -- saving rejects to file muc.lib.lua.rej
```

What can I do? 

EDIT:
If I comment out the following line in prosody Dockerfile, the image builds fine, but I don't know if this is essential or not...

`patch -d /usr/lib/prosody/modules/muc -p0 < /prosody-plugins/muc_owner_allow_kick.patch && \`

**Comment by saghul** [comment]
I guess that's because you got Prosody 0.12 which was just released. Without that patch a moderator won't be able to kick another moderator.

We'll look into it.

**Comment by saghul** [comment]
We have @aaronkvanmeerten looking into it. Alas it looks like the update can be a bit hairy, so we may pin the version in the interim.

**Comment by saghul** [comment]
I pushed a fix to master to pin the version to 0.11.

**Comment by pandel** [comment]
Many thanks, it is working. I close this issue for now.
